framework/src/test/java/com/epam/ta/test/CommonConditions.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.ta.test;
+
+import com.epam.ta.driver.DriverSingleton;
+import com.epam.ta.page.CloudPricingCalculatorPage;
+import com.epam.ta.page.TenMinuteMailHomePage;
+import com.epam.ta.util.TestListener;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Listeners;
+
+@Listeners({TestListener.class})
+public class CommonConditions {
+
+    WebDriver driver;
+    CloudPricingCalculatorPage cloudPricingCalculatorPage;
+    TenMinuteMailHomePage tenMinuteMailHomePage;
```

Comment:
private CloudPricingCalculatorPage cloudPricingCalculatorPage;
private TenMinuteMailHomePage tenMinuteMailHomePage;

Reply:
It's used in a subclass GoogleCloudEmailTest.